Comp2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -167,14 +174,14 @@ private void configureButtonBindings( )
     // Driver Controller Assignments
     //
     // Driver - A, B, X, Y
-    // m_driverPad.a( ).onTrue(new Dummy("driver A"));                      // TODO: temporarily used for CTRE testing
-    // m_driverPad.b( ).onTrue(new Dummy("driver B"));                   // TODO: temporarily used for CTRE testing
-    m_driverPad.x( ).onTrue(new Dummy("driver X"));
-    // m_driverPad.y( ).onTrue(drivetrain.driveToSpeaker(drivetrain));
+    m_driverPad.a( ).onTrue(new Dummy("driver A"));
```

Comment:
I don't think this matches the latest XBox controls spreadsheet. Close, but the driving buttons are on left bumper/trigger, and there are other controls on the right now.